### ISSUE TITLE
Let users choose where to install the app

### ISSUE THREAD

**Issue by Xendergo** [itchio/itch#2077]
I have two drives, a C and D drive, and the C drive is completely full and the D drive is mostly empty, and I can't combine the drives, so I can't install Itch because it has to write to the completely full drive. Heres the log file: 
[SquirrelSetup.log](https://github.com/itchio/itch/files/2343672/SquirrelSetup.log)

**Comment by notNSANE** [comment]
this is partly doable now on latest kitch @F4Tornado, with should be stable pretty soon :)

That said, all user preferences are still stored on %appdata% folder, sadly. 

I suggested a portable mode ages ago and Squirrel is/was the issue. I also suggested Squirrel devs to allow install in a drive other than C: and they completely denied that request.

https://github.com/itchio/itch/issues/650

Not sure what changed now, but at least itch can be installed somewhere else.
